### PR TITLE
[Fix Issue #15683] `readonly` input should not have a not-allowed cursor

### PR DESCRIPTION
**Pull request by kkirsche**
[Fix Issue #15683] `readonly` input should not have a `not-allowed` cursor. Instead, this pull request displays a standard cursor and only displays the not-allowed cursor on disabled items.

### CODE REVIEW

**Comment by cvrebert** [comment]
:+1: from me

**Comment by mdo** [comment]
:shipit:

**Comment by mdo** [comment]
One quick thought though—let's check the docs to see if there's anything we need to update language-wise for referring to the cursors of `readonly` and `disabled` inputs.

**Comment by kkirsche** [comment]
I can modify the text on disabled state (which currently is, "Add the `disabled` boolean attribute on an input to prevent user input and trigger slightly different looks") to include that it changes the cursor as well. E.g. "Add the `disabled` boolean attribute on an input to prevent user input and trigger slightly a muted color scheme with a `not-allowed` cursor"

Minor changes to the readonly section may be necessary. Currently, it is "Add the `readonly` boolean attribute on an input to prevent user input and style the input as disabled" but I would propose "Add the `readonly` boolean attribute on an input to prevent user input and style the input field as disabled while retaining the standard cursor"

Thoughts?

**Comment by mdo** [comment]
@kkirsche Yeah, I can dig that, and that sounds like it'd address both changes clearly enough. I'd suggest going with the following copy though:

> Add the `disabled` boolean attribute on an input to prevent user interactions. Disabled inputs appear lighter and add a `not-allowed` cursor.
>
> Add the `readonly` boolean attribute on an input to prevent modification of the input's value. Read-only inputs appear lighter (just like disabled inputs), but retain the standard cursor.

Is that too wordy, @kkirsche and @cvrebert?

**Comment by kkirsche** [comment]
No, I think that works well. If @cvrebert agrees I'll make those changes :+1: Thanks for the feedback

**Comment by cvrebert** [comment]
SGTM.

**Comment by kkirsche** [comment]
Updated. Thank you both for the feedback. I took the liberty of linking the disabled input text on the `readonly` to that section, hope you don't mind. If you do, I can take that out though, I just thought it might be helpful to easily get there if you wanted to see the difference. :+1:

**Comment by kkirsche** [comment]
Squashed commits into single due to small size of this change and it seems to have stabilized in regards to feedback.

**Comment by kkirsche** [comment]
Just wanted to follow up on this and see if anything was missing.

**Comment by kkirsche** [comment]
Thanks guys!
